src/status_im2/common/home/actions/view.cljs

```diff
@@ -175,13 +185,35 @@
 
 (defn edit-nickname-entry
   [chat-id]
-  (entry {:icon                :i/edit
-          :label               (i18n/label :t/edit-nickname)
-          :on-press            #(edit-nickname-action chat-id)
-          :danger?             false
-          :accessibility-label :edit-nickname
-          :sub-label           nil
-          :chevron?            false}))
+  (let [{:keys [nickname public-key secondary-name]
+         :as   contact} (select-keys (rf/sub [:contacts/contact-by-address chat-id])
+                                     [:primary-name :nickname :public-key :secondary-name])
+        no-nickname?    (string/blank? nickname)]
+    (entry
+     {:icon                (if no-nickname?
+                             :i/edit
+                             :i/delete)
+      :label               (i18n/label (if no-nickname?
+                                         :t/add-nickname-title
+                                         :t/remove-nickname))
+      :on-press            (fn []
+                             (if no-nickname?
+                               (edit-nickname-action contact)
+                               (do
+                                 (rf/dispatch [:hide-bottom-sheet])
+                                 (rf/dispatch [:toasts/upsert
+                                               {:id         :remove-nickname
+                                                :icon       :correct
```

Comment:
The Icon name should have the `:i/` qualification as per the guideline.
https://github.com/status-im/status-mobile/blob/develop/doc/new-guidelines.md#icons

Reply:
Fixed